src/framework/models/exceptions.kt

```diff
@@ -24,3 +24,10 @@ class InvalidArguments(private var body: String) :
  */
 class RouterException(private var resource: String) :
         MyException(404, "The route/resource $resource doesn't exist")
+
+/**
+ * Not Found Exception
+ * @constructor The default exception is 404 not found
+ */
+class NotFoundException :
```

Comment:
I would add an optional ability to pass a string

so

class NotFoundException(val message: String? = "Not Found")